how-to/misc/adding-amd-sev-enabled-compute-node.rst

```diff
@@ -0,0 +1,71 @@
+Adding AMD SEV enabled Compute Node
+===================================
+
+Secure Encrypted Virtualization is a technology from AMD enabling encryption of guest memory.
+OpenStack provides support to make use of this technology and deploy trusted guests.
+Refer `Admin guide <https://docs.openstack.org/nova/latest/admin/sev.html>`_ for more information.
+
+In Canonical OpenStack, AMD SEV enabled compute nodes can be added using the :doc:`cluster scale out procedure </how-to/operations/scaling-the-cluster-out>` by adding compute role to the node.
+Canonical OpenStack auto detects the compute node if the node is AMD SEV enabled or not.
+
+For AMD SEV enabled compute nodes, sufficient memory need to be reserved for the host since SEV enabled guests memory pages are pinned in RAM.
+To set the reserved memory for the host, update manifest with the following configuration for openstack-hypervisor charm.
+
+::
+
+    core:
+      config:
+        software:
+          charms:
+            openstack-hypervisor:
+              config:
+                reserved-host-memory-mb-for-sev: 8192
+
+
+For manual bare metal provider, pass the updated manifest in join command
+
+::
+
+    cat TOKEN_FILE | sunbeam cluster join --manifest MANIFEST_FILE --role ROLES -
+
+For MAAS provider, pass the updated manifest in deploy command
+
+::
+
+    sunbeam cluster deploy --manifest MANIFEST_FILE
+
+The configuration will be applied only on AMD SEV enabled compute nodes.
+
+.. caution::
+   **Note:** This feature requires snapd 2.68.x and above which is available in latest/candidate.
```

Comment:
It looks like 2.68.4 was released yesterday. I think we can remove this admonishment